src/components/view.js

```diff
@@ -1,35 +1,98 @@
-import { state, SUGGESTION_LIST, RECENT_SEARCHES_LIST } from '../index';
+import { state } from '../index';
+import {
+  suggestionList,
+  recentSearchList,
+  propertyList,
+  matchesCount,
+  loadMoreButton,
+} from './domConsts';
 
-export async function renderSuggestions() {
-  while (SUGGESTION_LIST.firstChild) {
-    SUGGESTION_LIST.removeChild(SUGGESTION_LIST.firstChild);
+export function renderSuggestions() {
+  const suggestionToShow = 4;
+  const suggestionHeight = 56;
+
+  while (suggestionList.firstChild) {
+    suggestionList.removeChild(suggestionList.firstChild);
   }
```

Comment:
Please move this to separate function, you can create file utils and import helpers from there